frontend/src/components/games/list/GameTable.tsx

```diff
@@ -91,9 +149,16 @@ export default function GameTable({
     contextMenu,
     limitFreeTier,
     columns,
+    defaultVisibility,
 }: GameTableProps) {
     const freeTierLimited = useFreeTier() && limitFreeTier;
     const { data, request, page, pageSize, rowCount, hasMore, setPage } = pagination;
+    const [columnVisibility, setColumnVisibility] = useState<GridColumnVisibilityModel>({
```

Comment:
If you're wanting to make these settings sticky, you can do it very easily by replacing the useState call with useLocalStorage

Reply:
Done. Added a namespace since different pages will want different columns visible (e.g. your own games versus other's)